openpype/hosts/maya/api/fbx.py

```diff
@@ -198,5 +203,6 @@ def export(members, path):
             path (str): Path to use for export.
 
         """
-        cmds.select(members, r=True, noExpand=True)
-        mel.eval('FBXExport -f "{}" -s'.format(path))
+        with maintained_selection():
+            cmds.select(members, r=True, noExpand=True)
+            mel.eval('FBXExport -f "{}" -s'.format(path))
```

Comment:
```suggestion
        # The export requires forward slashes because we need
        # to format it into a string in a mel expression
        path = path.replace("\\", "/") 
        with maintained_selection():
            cmds.select(members, r=True, noExpand=True)
            mel.eval('FBXExport -f "{}" -s'.format(path))
```